values.yaml

```diff
@@ -92,6 +92,39 @@ tolerations: []
 affinity: {}
 
 
+# ----------------------------------------------------------------------
+# DEPLOYMENT CONFIGURATION
+# ----------------------------------------------------------------------
+frontend:
+  replicas: 1
+  image:
+    repository: frontend
+    tag: develop
+  service:
+    type: ClusterIP
+    port: 80
+  app_env: production
+  ingress:
+    annotations:
+
+api:
+  replicas: 1
+  image:
+    repository: farmdoc-api
+    tag: develop
+  service:
+    type: ClusterIP
+    port: 5000
+  ingress:
+    annotations:
+  env:
+    postgres_user: postgres
+    postgres_password: postgres
+    postgres_host: farmdoc-postgresql-hl
```

Comment:
we can leave these blank, same for port etc below.

Reply:
made postgres_user, ostgres_password, and port as blank